iOS Example Frame/iOS Example Frame/ViewController/HomeViewController.swift

```diff
@@ -170,12 +165,50 @@ class HomeViewController: UIViewController {
 
 extension HomeViewController: ThreedsWebViewControllerDelegate {
   func threeDSWebViewControllerAuthenticationDidSucceed(_ threeDSWebViewController: ThreedsWebViewController, token: String?) {
-      threeDSWebViewController.dismiss(animated: true, completion: nil)
-      showAlert(with: "3DS success, token: \(token ?? "nil")")
+    threeDSWebViewController.dismiss(animated: true, completion: nil)
+    showAlert(with: "\(token ?? "nil")", title: "3DS success")
   }
 
   func threeDSWebViewControllerAuthenticationDidFail(_ threeDSWebViewController: ThreedsWebViewController) {
-      threeDSWebViewController.dismiss(animated: true, completion: nil)
-      showAlert(with: "3DS Fail")
+    threeDSWebViewController.dismiss(animated: true, completion: nil)
+    showAlert(with: "3DS Fail")
+  }
+}
+
+extension HomeViewController: PKPaymentAuthorizationViewControllerDelegate {
+
+  func paymentAuthorizationViewControllerDidFinish(_ controller: PKPaymentAuthorizationViewController) {
+    dismiss(animated: true, completion: nil)
```

Comment:
```suggestion
    dismiss(animated: true)
```

interesting that Apple wanted to make this callback compulsory, but not getting the payment authorisation. Suppose they want app developer to control when to dismiss screen, so potentially you can get a free loading overlay whilst processing the token? 🤔